azure/packages/azure-client/package.json

```diff
@@ -37,7 +37,7 @@
 		"test:realsvc:tinylicious": "start-server-and-test start:tinylicious:test 7070 test:realsvc:local:run",
 		"tsc": "tsc",
 		"typetests:gen": "fluid-type-test-generator",
-		"typetests:prepare": "flub generate typetests --prepare --dir . --pin"
+		"typetests:prepare": "flub typetests --dir . --reset --previous --normalize"
```

Comment:
Hmmm, maybe these were reverted in a main/next integrate at some point. I could have sworn they were updated at one point.